worker/evCheckerWorker/evCheckerWorker.go

```diff
@@ -9,29 +9,36 @@ import (
 	"io/ioutil"
 	"os/exec"
 	"reflect"
+	"os"
 )
 
 var workerName = "evCheckerWorker"
-var workerDesc = `Provides insights for EV certificates`
+var workerDesc = `Determines if a given EV policy fulfills the requirements of Mozilla's Root CA program.`
 
 var log = logger.GetLogger()
 
 func init() {
+	if _, err := os.Lstat(EvCheckerBinaryName); err != nil {
```

Comment:
Sure, but I thought you said we should just take the location of the binary as a config option